client/packages/common/src/intl/locales/en/dispensary.json

```diff
@@ -1,98 +1,99 @@
 {
-  "button.new-patient": "New Patient",
+  "button-save-as-visited": "Save as Visited",
```

Comment:
Sorry... everything got formatted 👀

Reply:
Don't know which git UI you are using but you can usually select only your changes and stage them and then revert unrelated stages...

I think related items where close together in this json file. However, we could agree on a json formatter everybody uses and then use a naming convention to group items, e.g. always start with `contact-trace-...` instead of `add-contact-trace-...`

Reply:
It's just ordering them in alphabetical order XD